### PR TITLE
Update readme for #76

### PR DESCRIPTION
**Pull request by lyp830414**
Signed-off-by: lyp830414 <lyp830414@163.com>

Modify read me for:
#76  Could not get script via 'curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash'

### CODE REVIEW

File: README_ZH.MD [hunk 1]
```diff
@@ -24,14 +24,43 @@ Probe是在[TWGC performance work group](https://github.com/Hyperledger-TWGC/fab
...
+1. 安装fabric-sample环境
+
+   `curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash`
+
+ **【注】**
+ ```
+  如果上述curl命令执行失败，这可能是由于部分国家或地区的网络原因导致curl的ssl握手失败。
+  在这种情况下，我们建议用户手工下载bootstrap.sh脚本和运行。
+ ```
+3. 下载tape的dockeer版本 `docker pull guoger/tape`
```
Comment (SamYuan1990) [inline README_ZH.MD:37]: 不要用 3.
用 1.

File: README_ZH.MD [hunk 2]
```diff
@@ -24,14 +24,43 @@ Probe是在[TWGC performance work group](https://github.com/Hyperledger-TWGC/fab
...
+
+6. 打开`localhost:3000`, 点击![TestNetworkSample](doc/quick_sample.png)
+
+7. 点击![submit](doc/quicksample2.png)
+
+8. 待完成后访问 `localhost:3000/result/BatchTimeout` 查看TPS与BatchTimeout之间的关系
+
+9. 待完成后访问 `localhost:3000/result/MaxMessageCount` 查看TPS与MaxMessageCount之间的关系
+
+10. 下载tape的dockeer版本 `docker pull guoger/tape`
```
Comment (SamYuan1990) [inline README_ZH.MD:51]: 重复？

File: README_ZH.MD [hunk 3]
```diff
@@ -24,14 +24,43 @@ Probe是在[TWGC performance work group](https://github.com/Hyperledger-TWGC/fab
...
-1. 待完成后访问 `localhost:3000/result/MaxMessageCount` 查看TPS与MaxMessageCount之间的关系
+
+1. 安装fabric-sample环境
+
+   `curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash`
+
+ **【注】**
+ ```
+  如果上述curl命令执行失败，这可能是由于部分国家或地区的网络原因导致curl的ssl握手失败。
+  在这种情况下，我们建议用户手工下载bootstrap.sh脚本和运行。
```
Comment (SamYuan1990) [inline README_ZH.MD:35]: 建议给出链接到https://github.com/hyperledger/fabric/blob/master/scripts/bootstrap.sh 这个文件的链接。
通过mark down 超链接格式

[bootstrap.sh](https://github.com/hyperledger/fabric/blob/master/scripts/bootstrap.sh)

**Review by SamYuan1990 — CHANGES_REQUESTED** [review]

**Comment by lyp830414** [comment]
All updated @SamYuan1990

**Review by SamYuan1990 — APPROVED** [review]

**Pull request merged**